src/main/groovy/cruise/umple/UmpleGradlePlugin.groovy

```diff
@@ -9,34 +9,62 @@ import org.gradle.api.GradleException
 
 class UmpleGradlePlugin implements Plugin<Project> {
 
-	private UmpleConsoleConfig consoleConfig 
-	private UmpleConsoleMain consoleMain
+	// Member variables
+	private String m_umpleFileName = ""
+	private String m_languageToGenerate = ""
+	private String m_outputPath = ""
+	private String m_defaultOutputPath = "../../libs/"
```

Comment:
Seeing a local path like this implies that this file knows where it will be run from, it doesn't. Don't put relative paths within code. Put "relative" paths if you know of environment variables. Never put paths that are outside a project unless you **know** that path exists. 

For example, on my system, this "../../libs" path actually tries to write to "/Users/libs" which it has no right/business doing.